assets/js/pages/profile.js

```diff
@@ -114,6 +114,9 @@ export default class ProfilePage {
     // track outbound links
     $('a[href^=http]').on('click', (e) => {
       ga('send', 'event', 'outbound-click', e.target.href);
+      gtag('event', 'outbound_click', {
+        target: e.target.href,
```

Comment:
let's put this in `label` instead of `action`

Reply:
Changed to label